source/deprecations/v1.x.html.md

```diff
@@ -349,6 +349,13 @@ making the content a `SafeString`. For example:
   })
 ```
 
+Make sure you don't put quotes around the string marked-as-safe when you bound
+it in the template, as this would prevent Ember from seeing it as safe:
+
+```handlebars
+<div style={{myStyle}}></div>
```

Comment:
This is exactly the same example as a few paragraphs above. Perhaps you can just add test to one of those paragraphs making this more explicit? Repeating the code doesn't seem helpful.